docs/MANPAGE_SYNTAX.md

```diff
@@ -29,45 +29,51 @@ podman\-command - short description
 **podman subcommand command** [*optional*] *value* [*value* ...]
 
 ## DESCRIPTION
-**podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the DESCRIPTION ensures uniformity. All commands mentioned in a text retain their appearance and form.\
+**podman command** is always the beginning of the DESCRIPTION section. Putting the command as the first part of the DESCRIPTION ensures uniformity. All commands mentioned in a text retain their appearance and form. Commands that are quoted from other manpages have to be linked to those. None podman commands aren't to be linked but should be listed under SEE ALSO. This ensures clarity.\
```

Comment:
I can't grok `None podman commands`. Is that supposed to be `Note`?

Reply:
OH! I think you mean "Non-podman commands" then? Is that it?

Reply:
Yes I believe it is.